### PR TITLE
Create no2-1

### PR DESCRIPTION
**Pull request by PuffYang**
自己浅显的了解了一下有关栈的知识，参考CSDN的代码写的（https://blog.csdn.net/yirenDM/article/details/118437119）

### CODE REVIEW

**Comment by chocolatedesue** [comment]
I see, thanks for hard working. 


I will test it tomorrow and inform you the result.

**Comment by chocolatedesue** [comment]
> 自己浅显的了解了一下有关栈的知识，参考CSDN的代码写的（[https://blog.csdn.net/yirenDM/article/details/118437119）](https://blog.csdn.net/yirenDM/article/details/118437119%EF%BC%89)

The base function is ok, but there are some small problems

1.  please follow  the file layout in this repo. One folder with .cpp and test data
2.  maybe the run menu can be vertical not horizontal, which contribute to a user-frendly experience.<br> **example**
 image

File: homework/no2-1 [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+#include<string.h>
+using namespace std;
+#define SIZE 1000//栈的大小
+//The first part:栈 
+struct stack{
+	int array[SIZE];
+	int top;//模拟指针,标记栈顶 
+}; 
+//初始化栈
+void initialize(stack* p){
```
Comment (chocolatedesue) [inline homework/no2-1:12]: Can be done with constructor, which is more standard

File: homework/no2-1 [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+		p->top++;
+	}
+}
+//出栈
+void pop(stack* p){
+     if(isEmpty(p)){
+     	cout<<"Error:The Empty Stack Can Not Be Ejected!"<<endl;
+	 }
+	 else{
+	 	printf("%d\n",p->array[p->top]);
```
Comment (chocolatedesue) [inline homework/no2-1:56]: need a prompt message

File: homework/no2-1 (hunk 1 above)
Comment (PuffYang) [reply homework/no2-1:12]: Sorry，I can't fix the problem at present for the lack of the knowledge about stack. Could you please provide the concrete codes of this part?

**Pull request closed without merging**